app/views/events/delete.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= link_to("<< Back to List", events_path, :class => 'back-link') %>
+
+<div class="book show">
```

Comment:
This seems like it is the wrong class for this situation: `class="book show"`

Reply:
Thanks for the catch! It's pretty easy to see where I got the code from